### PR TITLE
TPP-00010

### LINKED ISSUE #10: TPP-00010: Period Settings Page
> As a user of Menstruation Nation, I want to be able to customize how the app works for me in the settings page.

**Technical Information:**

- Add a section called "Your Stats" with two cards that have average cycle length and average period length. 
  - **NOTE:** keep the numbers displayed in avg cycle length and avg period length as placeholders for now
- Add a section called "Tracking Preferences" with 5 square buttons. They do not have to be interactable at this stage.
- Add a section called "Notifications" with two cards, and a switch on each card.
  - **NOTE:** you do not need to add the right arrow on these cards. you also do not need to make it interactable.

**Styling Requirements:**

The page does not need to be completely styled. You may leave the components on the entire page with their default designs. This means that not all fonts/colors/sizes have to be consistent with the Figma design.

image

[Figma Link](https://www.figma.com/file/YpKp60yVbVi4fEcQcQRoX3/TPP-User-Testing?node-id=94%3A9338) 

**Good Resources:**
[React Native Grid (For the "Tracking Preferences")](https://github.com/yelled3/react-native-grid-example)

### PR DESCRIPTION
**Pull request by kenneth-miura**
# Description

Added and styled specified components for specified issues.

Also added Podfile.lock to .gitignore

Fixes #10 
image

## Type of change 

- [x] New feature (non-breaking change which adds functionality)

# Questions
1. I don't think I need to comment this code b/c it's all structural basically. Do I?

# Checklist:

- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my own code
- [x] I have commented my code, particularly in hard-to-understand areas
- [x] I have made corresponding changes to the documentation (if required)
- [x] My changes generate no new warnings
- [x] I have added tests that prove my fix is effective or that my feature works

### CODE REVIEW

File: tpp-app/src/settings/Settings.js [hunk 1]
```diff
@@ -1,18 +1,160 @@
...
+                toggleSymptomsSwitch={toggleSymptomsSwitch}
+            />
+        </View>
+    )
+}
+
+const styles = StyleSheet.create({
+    rowContainer: {
+        flexDirection: 'row',
+        justifyContent: 'space-between'
```
Comment (yuemily24) [inline tpp-app/src/settings/Settings.js]: Try adding an attribute `alignItems` to vertically center the text and switch with each other. ([attribute description here](https://css-tricks.com/snippets/css/a-guide-to-flexbox/#align-items))

**Review by yuemily24 — CHANGES_REQUESTED** [review]
Code looks clean! Just one small style change requested.
Re comments: I don't think comments are necessary here for the reason you listed.

**Review by helen-li — APPROVED** [review]
Super organized! All looks good other than the centering styling Emily mentioned. 
PS: Regarding your question, I also don't think comments are necessary here.

**Review by yuemily24 — APPROVED** [review]
Looks good!

**Pull request merged**